packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -1572,7 +1572,7 @@ export function attach(
       if (nextPrimaryChildSet !== null) {
         mountFiberRecursively(
           nextPrimaryChildSet,
-          nextFiber,
+          shouldIncludeInTree ? nextFiber : parentFiber,
```

Comment:
Huh. Looks like we handled this correctly in 5 of the 6 places, but missed this one. Great find! :grin:

Reply:
Thank you!